torch_xla/csrc/xla_sharding_util.h

```diff
@@ -151,15 +151,16 @@ class ShardingUtil {
       const std::vector<std::string>& devices,
       const XLATensor::ShardingSpecPtr& sharding_spec);
 
-  static void xla_mark_sharding(const at::Tensor& input,
-                                xla::OpSharding sharding);
+  //////////////////////////// Dynamo Integration ////////////////////////////
+
+  static void XlaMarkSharding(const at::Tensor& input,
```

Comment:
nit: the `XlaMarkSharding` function itself here is not actually part of Dynamo Integration as the comment suggests, right? Only the `XlaMarkShardingDynamoCustomOp` is part of Dynamo Integration.

Reply:
SGTM